server/src/main/java/com/netflix/conductor/bootstrap/Main.java

```diff
@@ -33,25 +33,26 @@
 
     public static void main(String[] args) throws Exception {
 
+
         BootstrapUtil.loadConfigFile(args.length > 0 ? args[0] : System.getenv("CONDUCTOR_CONFIG_FILE"));
 
         if (args.length == 2) {
             System.out.println("Using log4j config " + args[1]);
             PropertyConfigurator.configure(new FileInputStream(new File(args[1])));
         }
 
-        Injector bootstrapInjector = Guice.createInjector(new BootstrapModule());
-        ModulesProvider modulesProvider = bootstrapInjector.getInstance(ModulesProvider.class);
-        Injector serverInjector = Guice.createInjector(modulesProvider.get());
+        final Injector bootstrapInjector = Guice.createInjector(new BootstrapModule());
+        final ModulesProvider modulesProvider = bootstrapInjector.getInstance(ModulesProvider.class);
+        final Injector serverInjector = Guice.createInjector(modulesProvider.get());
 
-        Optional<EmbeddedElasticSearch> embeddedElasticSearch = serverInjector.getInstance(EmbeddedElasticSearchProvider.class).get();
+        final Optional<EmbeddedElasticSearch> embeddedElasticSearch = serverInjector.getInstance(EmbeddedElasticSearchProvider.class).get();
         embeddedElasticSearch.ifPresent(BootstrapUtil::startEmbeddedElasticsearchServer);
 
         BootstrapUtil.setupIndex(serverInjector.getInstance(IndexDAO.class));
 
         try {
             serverInjector.getInstance(IndexDAO.class).setup();
-        } catch (Exception e) {
+        } catch (final Exception e) {
             e.printStackTrace(System.err);
```

Comment:
should use proper logging here, and not print stack trace

Reply:
I didn't write the code in main class. That was there from Netflix

Reply:
bad netflix